include/struct_defs/struct_02057B48.h

```diff
@@ -2,12 +2,12 @@
 #define POKEPLATINUM_STRUCT_02057B48_H
 
 typedef struct {
-    u16 unk_00;
-    u16 unk_02;
-    s8 unk_04;
-    u8 unk_05;
+    u16 x;
+    u16 z;
```

Comment:
```suggestion
    u16 y;
```

Reply:
as we have established in the hgss repo, we use the cringe z-up notation due to the orientation of (x, y, z) vs (x, z, y) causing the vector math to be different

Reply:
As cringe as it is, it's just how the DS assumes planes are rendered, to my knowledge. Vector math in the Pokemon games, at least, is contingent on the player running along the XZ plane (and Platinum specifically makes this important with the Distortion World).